controllers/backupcontroller.py

```diff
@@ -1,14 +1,4 @@
-from TestData import*
-from Controller import*
-class BackupController(Controller):
+class BackupController():
+  testData
   def __init__(self):
-      self.testData = TestData()
```

Comment:
I suspect removing line 5 was a mistake as self.testData must be initialized before line 4 can be run. At this time it will raise a fatal exception at runtime.